### ISSUE TITLE
Provenance 1.2.3 (current as of 2015-11-05) fails to import Sega CD Roms

### ISSUE THREAD

**Issue by brianmcdoogle** [jasarien/Provenance#180]
Provenance 1.2.3 (current as of 2015-11-05) fails to import Sega CD Roms

I recently set up a new phone from scratch and the importing of Sega CD roms doesn't currently work for me. I am using the same rom and cue files I used on a previous phone this summer (I'm sorry I can't recall the version number or Provenance, but it would have been an 8.3 or 8.4 iOS on an iPhone 5S) and the same BIOS files using the same method outlined in the importing wiki. However, importing the zip containing the bin and cue files always fails on extraction (the app will hard quit), and reopening the app will cause another import, but the rom this time will be listed as both a Sega CD and Genesis game. And then when I attempt to open the rom, I get an error that the rom cannot be loaded (Either tapping the Genesis or Sega CD duplicate entries). I've tried doing this several times, but it happens every time, and I continue to get errors if I delete the bogus Genesis entry that gets added.

This is on Provenance 1.2.3 (current version as of today, 2015-11-05) on a 6S Plus running iOS 9.1. And, again, this was working on a previous version of Provenance that was a few months old on an older phone, and the bin/cue files will work on emulators on both Mac and PC.

**Comment by jasarien** [comment]
Hm, strange. I'm sure I fixed that issue in this commit: https://github.com/jasarien/Provenance/commit/a6be14f1003215a817791bd1fb9c1d482f68db32

I also tested on my iPhone with 1.2.3 installed just now and it all seemed to work fine: http://d.pr/v/5Nb5

Can I suggest that you delete Provenance and reinstall it from scratch and see if this still happens?

**Comment by brianmcdoogle** [comment]
I tried a few reinstalls and got the same problem over and over, so I tried another method of adding the files to the iPhone and it looks like the program I was using was adding a space to the front of files, so the name inside the cue file wasn't matching the bin file name because the bin file had a space in front of it, and the cue file must have the exact same name as the bin, as well as the file name inside of the cue. So, problem was with that app and not Provenance. All is well!
